src/consolebundle/CommandManager.py

```diff
@@ -9,9 +9,40 @@ def __init__(self, commands: List[ConsoleCommand]):
     def get_commands(self):
         return self.__commands
 
-    def get_by_name(self, name: str) -> ConsoleCommand:
+    def log_commands(self, logger):
```

Comment:
I would propose to move log_commands() and log_subcommands() out of CommandManager...probably to CommandRunner

Reply:
Ok, moved.